index.html

```diff
@@ -50,46 +50,36 @@
   <!-- End Matomo Code -->
 </head>
 <body>
-	<nav>
-		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
-			<span class="newsletter-date">July-October, 2019</span>
-      <a class="language-menu--item" href="/" itemprop='url'>
-        <span itemprop="name">English</span>
-      </a>
-      <a class="language-menu--item" href="/es/" itemprop='url'>
-        <span itemprop="name">Español</span>
-      </a>
-		</div>
-    <div class="primary-nav" itemscope itemtype="http://schema.org/Organization">
-      <a class="logo-btn" href="/" itemprop="url">
+  <header>
+  	<nav itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
+  		<div class="menu">
+        <span class="menu__news-name">MACILE Newsletter</span>
+        <div class="wrapper">
```

Comment:
There aren't any css styles targeting the `wrapper` class so to avoid confusion can we remove it or follow BEM naming and tie the wrapper to it's block parent element? 

I vote to remove the class.